MyServlet/build.gradle

```diff
@@ -0,0 +1,36 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'org.akhikhl.gretty:gretty:+'
+    }
+}
+
+plugins {
+    id 'java'
+}
+apply plugin: 'war'
+apply plugin: 'org.akhikhl.gretty'
+group 'HelloServlet'
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+}
+
+dependencies {
```

Comment:
why do we need two dependencies blocks?

Reply:
Yeap, and with two the same dependencies at the first glance